code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -682,3 +572,94 @@ It'll return null if the organ doesn't correspond, so include null checks when u
 	if(!(organ_slot in has_organ))
 		return null
 	return has_organ[organ_slot]
+
+
+/datum/species/proc/update_sight(mob/living/carbon/human/H)
+	if(!H.client)
+		return 1
```

Comment:
Isn't this redundant when species datums can only be attached to a human mob and it already checks for `!client` in human `update_sight`?